Add distribution page and data inside Ecosystem (#3854)
open-telemetry/opentelemetry.io: content/en/ecosystem/distributions.md

---
title: Distributions
description:
  Libraries, services, and apps with first-party support for OpenTelemetry.
aliases: [/distributions]
---

OpenTelemetry [distributions](/docs/concepts/distributions/) are a way of
customizing OpenTelemetry components so that they're easier to deploy and use
with specific observability back ends.

Any third-party can customize OpenTelemetry components with back-end, vendor, or
end-user specific changes. The following list contains a sample of OpenTelemetry
distributions and their type.

> 

Reviewer: Add a sentence here that calls out specifically, that a distribution is not a "must have", and that many organisations and vendors just live fine without one (especially for language SDKs, collector might be the exception here)

Author: How about this? https://github.com/open-telemetry/opentelemetry.io/pull/3854/commits/9dff456af19688f51b2fbff9f45323a94624a239

{{% ecosystem/distributions-table %}}

## How to add your distribution {#how-to-add}

To have your library, service, or app listed, submit a PR with an entry added to
the
[distributions list](https://github.com/open-telemetry/opentelemetry.io/tree/main/data/ecosystem/distributions.yaml).
The entry must include the following:

> - Link to the main page of your distribution
> - Link to the documentation that explains how to use the distribution

Reviewer: ```suggestion
- Link to the main page of your distribution
- Link to the documentation that explains how to use the distribution
- List the components your distribution contains
- GitHub handle or email address as a point of contact so that we can reach out in case we have questions
```

---
title: Distributions
description: List of OpenTelemetry distributions maintained by third parties.
---

OpenTelemetry [distributions](/docs/concepts/distributions/) are a way of
customizing OpenTelemetry components so that they're easier to deploy and use
with specific observability backends.

Any third-party can customize OpenTelemetry components with back-end, vendor, or
end-user specific changes. You don't have to use a distributions in order to use
OpenTelemetry components, though distributions might facilitate usage under
certain circumstances, such as specific vendor requirements.

The following list contains a sample of OpenTelemetry distributions and the
component they customize. Note that those are third-party projects and are in

> no way validated or endorsed by the OpenTelemetry community.

Reviewer 2: I'd suggestion putting this note inside a `{{% alert title="Note" color="warning" %}}` possibly with revisted prose like this:

> OpenTelemetry **_does not_ validate or endorse** the third-party distributions listed below. The list is provided as a convenience for the community.

{{% ecosystem/distributions-table %}}

## How to add your distribution {#how-to-add}

> To have your library, service, or app listed, submit a PR with an entry added to
> the

Reviewer 2: ```suggestion
To have your distribution listed, submit a PR with an entry added to
the
```

[distributions list](https://github.com/open-telemetry/opentelemetry.io/tree/main/data/ecosystem/distributions.yaml).
The entry must include the following:

- Link to the main page of your distribution
- Link to the documentation that explains how to use the distribution
- List the components your distribution contains
- GitHub handle or email address as a point of contact so that we can reach out
  in case we have questions

> {{% alert title="Note" color="info" %}}
> 
> If you provide external integration of OpenTelemetry for any kind of library,
> service, or app, then
> [consider adding it to the registry](/ecosystem/registry/adding).
> 
> If you adopt OpenTelemetry for Observability as an end user and do not provide
> any kind of services around OpenTelemetry, see [Adopters](/ecosystem/adopters).
> 
> If you provide a solution that consumes OpenTelemetry to offer observability to
> end users, see [Vendors](/ecosystem/vendors).

Reviewer 2: ```suggestion
{{% alert title="Notes" color="info" %}}

- If you provide external integration of OpenTelemetry for any kind of library,
service, or app, then consider
[adding it to the registry](/ecosystem/registry/adding).

- If you adopt OpenTelemetry for observability as an end user and do not provide
any kind of services around OpenTelemetry, see [Adopters](/ecosystem/adopters).

- If you provide a solution that consumes OpenTelemetry to offer observability to
end users, see [Vendors](/ecosystem/vendors).
```

---
title: Distributions

> description:
>   List of OpenTelemetry distributions maintained by third parties.

Reviewer 2: How about emphasizing OSS here?

```suggestion
description: >-
  List of open source OpenTelemetry distributions maintained by third parties.
```

Author: Added!

>   Libraries, services, and apps with first-party support for OpenTelemetry.

Reviewer: fix that description

> aliases: [/distributions]

Reviewer 2: ```suggestion
```

Reviewer 2: Hmm, this was marked as resolved, but the alias is still there.

Author: Hmm, let me check that

Author: Ah, now: https://github.com/open-telemetry/opentelemetry.io/pull/3854/commits/27604c5b125bad84c4b55f13aa29a929f6e63405

---

OpenTelemetry [distributions](/docs/concepts/distributions/) are a way of
customizing OpenTelemetry components so that they're easier to deploy and use
with specific observability back ends.

Any third-party can customize OpenTelemetry components with back-end, vendor, or
end-user specific changes. You don't have to use a distributions in order to use
OpenTelemetry components, though distributions might facilitate usage under
certain circumstances, such as specific vendor requirements.

> The following list contains a sample of OpenTelemetry distributions and the
> component they customize.

Reviewer: ```suggestion
The list below contains a sample of OpenTelemetry distributions and the
component they customize. Note that those are third-party projects and are in no way validated or endorsed by the OpenTelemetry community.
```

Author: Edited your suggestion to change "below" to "following" as per https://developers.google.com/style/word-list#below

certain circumstances, such as specific vendor requirements.

The following list contains a sample of OpenTelemetry distributions and the
component they customize.

{{% alert title="Note" color="warning" %}} OpenTelemetry **does not validate or
endorse** the third-party distributions listed in the following table. The list
is provided as a convenience for the community. {{% /alert %}}

{{% ecosystem/distributions-table %}}

## How to add your distribution {#how-to-add}

To have your distribution listed, submit a PR with an entry added to the
[distributions list](https://github.com/open-telemetry/opentelemetry.io/tree/main/data/ecosystem/distributions.yaml).
The entry must include the following:

- Link to the main page of your distribution
- Link to the documentation that explains how to use the distribution
- List the components your distribution contains
- GitHub handle or email address as a point of contact so that we can reach out
  in case we have questions

{{% alert title="Notes" color="info" %}}

- If you provide external integration of OpenTelemetry for any kind of library,
service, or app, then consider
[adding it to the registry](/ecosystem/registry/adding).

> 

Reviewer 2: ```suggestion
```

- If you adopt OpenTelemetry for observability as an end user and do not provide
any kind of services around OpenTelemetry, see [Adopters](/ecosystem/adopters).

> 

Reviewer 2: ```suggestion
```
